app/views/rating_questions/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="form">
+  <div class="flash-notice"><%= flash[:notice] %>
```

Comment:
Delete this from here, and have it exist within the application layout.

Reply:
done